appservice/src/createAppService/SiteRuntimeStep.ts

```diff
@@ -3,18 +3,35 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { WebResource } from 'ms-rest';
+import * as request from 'request-promise';
 import { workspace } from 'vscode';
 import { AzureWizardPromptStep, IAzureQuickPickItem } from 'vscode-azureextensionui';
 import { ext } from '../extensionVariables';
 import { localize } from '../localize';
+import { signRequest } from '../signRequest';
 import { AppKind, WebsiteOS } from './AppKind';
 import { IAppServiceWizardContext } from './IAppServiceWizardContext';
 
 interface ILinuxRuntimeStack {
```

Comment:
I don't like having `ILinuxRuntimeStack` _and_ the type in `availableStacksJson`. Can we consolidate? Would prefer the same type as the portal